python/ray/_private/worker.py

```diff
@@ -683,6 +683,7 @@ def get_objects(self, object_refs: list, timeout: Optional[float] = None):
             debugger_breakpoint,
         )
 
+    @Deprecated
```

Comment:
can we add some comments explain why it's deprecated?  (as what you put in the PR summary)

Reply:
Sure!